modules/API/API.js

```diff
@@ -356,6 +357,11 @@ function initCommands() {
         },
         'kick-participant': participantId => {
             APP.store.dispatch(kickParticipant(participantId));
+        },
+        'overwrite-configs': configs => {
+            const whitelistedConfig = getWhitelistedJSON('config', configs);
+
+            APP.store.dispatch(overwriteConfigs(whitelistedConfig));
```

Comment:
```suggestion
            APP.store.dispatch(overwriteConfig(whitelistedConfig));
```

Reply:
Let's go with "overwrite config" since we have "set config" and "update config" and not "configS"